cypress/e2e/pages/testsForAllAMPPages.js

```diff
@@ -7,12 +9,45 @@ export const testsThatFollowSmokeTestConfigForAllAMPPages = ({
   pageType,
 }) => {
   describe(`testsThatFollowSmokeTestConfigForAllAMPPages to run for ${service} ${pageType}`, () => {
+    describe(
+      'Image Tests',
+      {
+        retries: 3,
```

Comment:
I found when running the tests that sometimes one of the image tests would fail but pass other times. Maybe to do with the lazy loading as it seemed to be not finding the image. The retry stops it being as flakey.